### PR TITLE
Import event from facebook

### PR DESCRIPTION
**Pull request by BearGuy**
This feature allows the user to import events from facebook, and renders a preview modal to display the information before submitting it. Submit is still not implemented, as well the current change breaks our previous form. @PfeifferH We'll have to talk about some changes to the `EventAddForm.jsx` to get this to function fully.

### CODE REVIEW

File: src/components/event/eventAdd/EventAddContainer.jsx [hunk 1]
```diff
@@ -12,7 +13,7 @@ const mapDispatchToProps = dispatch => {
   return ({
     toggleCreateEventForm: () => {
       dispatch({ type: TOGGLE_CREATE_EVENT_HIDDEN })
-    }
+    },
```
Comment (PfeifferH) [inline src/components/event/eventAdd/EventAddContainer.jsx]: May be nitpicking here but I do not see the need for the extra comma

File: src/layouts/index.js [hunk 2]
```diff
@@ -3,8 +3,6 @@ import PropTypes from 'prop-types'
 
 import Header from '../components/header/HeaderContainer.jsx'
 import Footer from '../components/footer/Footer.jsx'
-import Login from '../components/auth/login/LoginContainer.jsx'
-import SignUp from '../components/auth/signUp/SignUpContainer.jsx'
```
Comment (PfeifferH) [inline src/layouts/index.js]: Where have the Login and Signup components implemented then?

File: src/components/event/eventAdd/EventAddContainer.jsx (hunk 1 above)
Comment (PfeifferH) [inline src/components/event/eventAdd/EventAddContainer.jsx]: May be nitpicking here but I do not see the need for the extra comma

Comment (BearGuy) [reply src/components/event/eventAdd/EventAddContainer.jsx]: I have it in there just so I don't forget to add it if I ever want to extend the functionality

File: src/layouts/index.js (hunk 2 above)
Comment (BearGuy) [reply src/layouts/index.js]: They're implemented in `AuthPopUp`

**Pull request merged**